.azure-pipelines.yml

```
@@ -81,7 +82,7 @@ jobs:
         openmpi-bin libopenmpi-dev \
         libfftw3-dev libfftw3-mpi-dev libhdf5-openmpi-dev pkg-config make \
         python3 python3-pandas python3-pip python3-venv python3-setuptools libblas-dev liblapack-dev
-      ccache --set-config=max_size=10.0G
+      ccache --set-config=max_size=2.0G
```

Single build with `-g` is 3.2 GB:
```suggestion
      ccache --set-config=max_size=5.0G
```